README.md

```diff
@@ -39,14 +39,14 @@ In order to use the new POM:
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>3.43</version> <!-- or whatever the newest version available is -->
+    <version>4.31</version> <!-- or whatever the newest version available is -->
     <relativePath />
   </parent>
 ```
 * Override the needed properties, e.g.:
 ```xml
   <properties>
-    <jenkins.version>2.60.1</jenkins.version>
+    <jenkins.version>2.204</jenkins.version>
```

Comment:
Should be mostly consistent with https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/

Reply:
Well, this is the current version specified in the pom.xml

Reply:
Right, but that is expected to be overridden. Here we are showing an example of what a plugin POM would normally look like, so
```suggestion
    <jenkins.version>2.289.1</jenkins.version>
```
or
```suggestion
    <jenkins.version>2.303.1</jenkins.version>
```

Reply:
> Here we are showing an example of what a plugin POM would normally look like

Agreed. On the other hand, I think it _is_ appropriate to use the minimum supported version in the Invoker tests (to ensure that plugin POM changes don't break support for older cores).

Reply:
> use the minimum supported version in the Invoker tests

In _at least one_ IT, that is. Some ITs specifically use some newer version because they need to, or just use some newer versions because we also want test coverage for the common case of a newish `jenkins.version`.